biz.aQute.bndlib/src/aQute/bnd/osgi/repository/BridgeRepository.java

```diff
@@ -327,6 +369,16 @@ public static void addInformationCapability(ResourceBuilder rb, String name, Ver
 				c.addAttribute("error", error);
 			if (version != null)
 				c.addAttribute("version", version);
+			if (groupId != null)
+				c.addAttribute("groupId", groupId);
```

Comment:
In the work I did to get the maven coordinate info into Container, I defined all the maven attr names to start with `maven-`:

https://github.com/bndtools/bnd/blob/7a5b449d1469cedefc10ecd8c7ec6292e91e6def/biz.aQute.repository/src/aQute/maven/api/Program.java#L165-L166

https://github.com/bndtools/bnd/blob/7a5b449d1469cedefc10ecd8c7ec6292e91e6def/biz.aQute.repository/src/aQute/maven/api/Revision.java#L131

https://github.com/bndtools/bnd/blob/7a5b449d1469cedefc10ecd8c7ec6292e91e6def/biz.aQute.repository/src/aQute/maven/api/Archive.java#L248-L249

I think we should use the same attr names here. Then is very clear where the names come from.

Reply:
Will this work via the proxy? Note that InfoCapability is created as a proxy, are you sure there is a mangling for those property names?

Reply:
Why not? You are adding attributes to a builder. You can do

```java
c.addAttributes(map);`
```

Reply:
it's not the map usage I'm concerned with, but your recommendation to use maven-* naming. Will our proxy handle those? I guess we could fix that too I suppose...